### ISSUE TITLE
Accessibility Insights for Windows reports an issue with MetroWindow

### ISSUE THREAD

**Issue by mrlacey** [MahApps/MahApps.Metro#3894]
## Describe the bug

The default behavior of the `MetroWindow` includes an issue that is reported as an accessibility problem and a potential violation of [Section 508 502.3.1](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/single-file-version#502-interoperability-assistive-technology).

## Steps to reproduce

1. Create a new WPF app (.NET Core)
2. Add a reference to MahApps.Metro
3. Change the Window (in MainWindow.xaml[.cs]) to be a `MetroWindow`
4. Launch the app
5. Live Inspect with [Accessibility Insights for Windows](https://accessibilityinsights.io/docs/en/windows/overview).

This is a minimal repro. The same error is reported if there is content specified and if LeftWindowCommands are specified.

## Expected behavior

There should be no accessibility issues reported for using this or any other control.

## Actual behavior

The unnamed "tool bar" is reported as showing twice in the UIA tree and in both instances, it does not have a bounding rectangle provided.

## Environment

```
MahApps.Metro version: v2.1.0
Windows build number: Win10 2004 [Version 10.0.19041.388]
Visual Studio: 2019 16.6.5
Target Framework: .Net Core 3.1 
```

## Screenshots

![image](https://user-images.githubusercontent.com/189547/89182010-b2cae300-d58c-11ea-8dbf-8d62a1bd711a.png)

X-Ref: https://github.com/microsoft/WindowsTemplateStudio/issues/3723
